ui/components/Filters/MesheryFilterTable.js

```diff
@@ -0,0 +1,133 @@
+import React from "react";
+import { Avatar, IconButton, TableCell, TableSortLabel } from "@mui/material";
+import MUIDataTable from "mui-datatables";
+import Moment from "react-moment";
+import DoneAllIcon from "@mui/icons-material/DoneAll";
+import EditIcon from "@mui/icons-material/Edit";
+import UndeployIcon from "../../public/static/img/UndeployIcon";
+
+function MesheryFilterTable({ filters = [], setSelectedRowData, handleModalOpen, user }) {
```

Comment:
@subhradwip01 You haven't added page, pagesize etc , i also didn't added it before, have a look at my latest PRs for reference